sdk/storage/azure-storage-file-share/tests/perfstress/download_to_file.py

```diff
@@ -0,0 +1,40 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import tempfile
+
+from ._test_base import _ShareTest
+
+
+class DownloadToFileTest(_ShareTest):
```

Comment:
Does this test add any value over `DownloadTest`?  It looks like the only difference is calling `stream.readall()` instead of `stream.readinto(fp)`, and we are probably not interested in measuring the perf difference between these two APIs, unless we have reason to believe the stream we return might have different perf in these scenarios.

The important thing here is we are calling the exact same SDK API, the only difference is what we do with the returned stream.  If the SDK provided an API to download directly to file, that might be more interesting to test.

Reply:
I see that Track1 did have an API to download directly to a file, so it probably does make sense to have an equivalent in Track2.

Reply:
Yeah - T1 had dedicated file APIs for upload and download - which we have removed and demonstrated with samples for handling files. So would be good to ensure we haven't removed perf in the process.